src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+$cell-size: 100px;
+$gap-size: 1px;
+$padding-size: 10px;
+$border-size: 1px;
+$calendar-width: (7 * $sell-size) + (6 * $gap-size);
```

Comment:
There is a typo in the variable name '$sell-size'. It should be '$cell-size' to match the defined variable on line 1. This typo will cause a compilation error.